### ISSUE TITLE
Debian fails to build due to `apt-get` errors

### ISSUE THREAD

**Issue by kBite** [systemd/mkosi#139]
Debian fails to build due to `apt-get` errors. Interestingly Ubuntu does not.

---
`mkosi.default`
```
[Distribution]
Distribution=debian
Release=jessie

[Packages]
WithNetwork=true
```
`# mkosi --default mkosi.default -o debian-8`
```
DISTRIBUTION:
          Distribution: debian
               Release: jessie
                Mirror: http://httpredir.debian.org/debian

OUTPUT:
         Output Format: raw_gpt
                Output: /root/mkosi/debian-8/debian-8

PACKAGES:
         Build Sources: /root/mkosi/debian-8
  Scripts with network: yes

PARTITIONS:
        Root Partition: 1.0G
```
```
WARNING: The following packages cannot be authenticated!
  libcap-ng0 libdbus-1-3 libexpat1 dbus libpam-systemd
E: There are problems and -y was used without --force-yes
```

---
`mkosi.default`
```
[Distribution]
Distribution=debian
Release=jessie

[Packages]
WithNetwork=true
```
`# mkosi --default mkosi.default -o debian-8`
```
DISTRIBUTION:
          Distribution: debian
               Release: jessie
                Mirror: http://httpredir.debian.org/debian

OUTPUT:
         Output Format: directory
                Output: /root/mkosi/debian-8/debian-8

PACKAGES:
         Build Sources: /root/mkosi/debian-8
  Scripts with network: yes
```
```
...
WARNING: The following packages cannot be authenticated!                           
  libcap-ng0 libdbus-1-3 libexpat1 dbus libpam-systemd                             
E: There are problems and -y was used without --force-yes 
...
```
`mkosi.default`
```
[Distribution]
Distribution=debian
Release=stretch

[Packages]
WithNetwork=true
```
`# mkosi --default mkosi.default -o debian-9`
```
DISTRIBUTION:
          Distribution: debian
               Release: stretch
                Mirror: http://httpredir.debian.org/debian

OUTPUT:
         Output Format: directory
                Output: /root/mkosi/debian-9/debian-9

PACKAGES:

         Build Sources: /root/mkosi/debian-9
  Scripts with network: yes
```
```
...
WARNING: The following packages cannot be authenticated!                           
  libdbus-1-3 libexpat1 dbus libpam-systemd                                        
E: There were unauthenticated packages and -y was used without --allow-unauthenticated
...
```
---
It's reproducible with
- latest `mkosi` from master 
- Ubuntu 16.04: `apt 1.2.24 (amd64)` and `debootstrap 1.0.90ubuntu1`
- Arch Linux (latest): `apt 1.4.6 (amd64)` and `debootstrap 1.0.91`

**Comment by kBite** [comment]
If I set `--allow-unauthenticated` like in #140 and start the resulting image via `nspawn`, `apt install wget` says `wget` is unauthenticated. But after `apt update`, it isn't unauthenticated anymore.

**Comment by fsateler** [comment]
Looks like the local copy of the debian archive keyring is outdated. I can't reproduce on debian unstable.

**Comment by kBite** [comment]
Damn, yes, that's right. After installing `debian-archive-keyring` everything works fine. Maybe the keyring should be added to the list of dependencies?

**Comment by fsateler** [comment]
Well, debootstrap recommends it:

```
% apt-cache show debootstrap | grep debian-archive
Recommends: gnupg, debian-archive-keyring
```

**Comment by kBite** [comment]
Not when installed on Ubuntu
```
# apt-cache show debootstrap | grep keyring
Recommends: gnupg, ubuntu-keyring
```

**Comment by poettering** [comment]
I added a doc fix explaining this in #163 now.
